spec/kn/checksums_spec.rb

```diff
@@ -0,0 +1,23 @@
+require './kn/checksums'
+
+RSpec.describe Kn::Checksums, '#sha256' do
+  context 'for kn-event v1.1.1' do
```

Comment:
Wrt adding new release workflow: a new context entry is always added to this file, e.g.  ` 'for kn-event v1.2.0' do`?

Reply:
I just wonder if it would be sufficient to compare checksum file with binary without additional maintenance of sha256 strings.

Reply:
No. This is a test for the Checksum class. No need to change anything here in the future.

Reply:
OK, thanks for the clarification.